docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -105,6 +135,12 @@ You'll need to replace the following variables in the `node-config.yaml` file to
 - `<HOSTNAME>`: Hostname for this node (e.g. kube-node1, kube-node2)
 - `<SSH_PUBLIC_KEY>`: The public key you will use for SSH access to this server.
 - `<KUBERNETES_MASTER>`: The IPv4 address of the Kubernetes master.
+- `<AUTH_TOKEN>`: The API authentication token generated in the previous step.
+
+Next, you will need to add the certificates generated in the previous step to the cloud-config. Replace the following placeholders with your TLS assests.
+- `<CA_CERT>`: Complete contents of `ca.pem`
+- `<WORKER_CERT>`: Complete contents of `worker.pem`
+- `<WORKER_KEY>`: Complete contents of `worker-key.pem`
 
 Move the modified `node-config.yaml` to your Kubernetes node machine and install and configure CoreOS on the node using the following command.
```

Comment:
Just out of interest, how does one do this? (And should it be copy rather than move?)